build.sbt

```diff
@@ -28,6 +28,7 @@ libraryDependencies ++=  List(
   "org.apache.spark"   %% "spark-core" % sparkVersion % "provided",
   "org.apache.spark"   %% "spark-sql" % sparkVersion % "provided",
   "za.co.absa.commons" %% "commons" % "1.0.0",
+  "com.typesafe"        % "config" % "1.4.1",
```

Comment:
I am not sure we want to depend on a singular properties implementation. Could we maybe pass in the properties somehow?

Reply:
To be noted that these properties need to be passed in `TimeZoneNormalizer` as well. I do not know of a nice general way to have this

Reply:
Maybe we will have to force users to create their own trait and this will be a builder more than anything else. With existing default trait.